ansible/bootstrap.yml

```diff
@@ -8,25 +8,31 @@
       - nomad
     token: master
   tasks:
+    - name: Check if .env_override is provided
+      debug:
+        msg: "We are running in CI and .env_override is set"
+      failed_when: local_test is undefined and not lookup( 'env', 'TF_VAR_env_override') | bool
+      tags: test
+
```

Comment:
```suggestion
    - name: Check if .env_override is provided
      debug:
        msg: "Checking if env_override process was successful. Doing nothing if this is a local test"
      failed_when: local_test is undefined and not lookup( 'env', 'TF_VAR_env_override') | bool
      tags: test

```

Reply:
Is this showing up as no changes for you as well?

Anyways, I've changed the message.